components/climate/bedjet.rst

```diff
@@ -0,0 +1,104 @@
+BedJet
+======
+
+.. seo::
+    :description: Instructions for setting up a BedJet climate device.
+    :image: bedjet.png
+
+The ``bedjet`` climate platform creates a climate device which can be used to control
+a BedJet V3 Climate Comfort Sleep System.
+
+This component supports the following functionality:
+
+- Set the operating mode: off, heat, cool, turbo (boost)
+- Set the desired target temperature
+- Set the desired fan speed
+- Start one of the saved memory presets, including "Biorhythm" programs
+- Show the current status of the BedJet
+
+This platform uses the BLE peripheral on an ESP32, so you also need to enable
+this component. Please see the BLE Client docs for how to discover the MAC
```

Comment:
```suggestion
this component. Please see the :doc:`/components/ble_client` docs for how to discover the MAC
```

Reply:
Done in 679f89d

Reply:
I think you force pushed over this and the commit is lost.

Reply:
I did force-push, but that was for a rebase. The current diff still shows your change is completed:
https://github.com/esphome/esphome-docs/pull/1508/files#diff-49c9c76ae2f8342da70cc8d5b97e9c6f43055b3b2995519ccea698fd510c8bceR20
And your commit is still in the history (now ed5515 after rebase): https://github.com/esphome/esphome-docs/pull/1508/commits

Reply:
Or I may be confused and maybe you added the commit back into my fork? 🤔  I just checked again, and I see it on my fork, but I did not see it locally, so I think you're right and I would have accidentally reverted the change when I pushed last.

But it's definitely there now, and I see it in my local copy as well, so now for sure I won't accidentally revert it if I rebase/merge again 👍